iina/Extensions.swift

```diff
@@ -321,6 +321,14 @@ extension Data {
   }
 }
 
+extension String {
```

Comment:
Alright, but I don't think we need to bridge to `NSString`/`NSData` here. Can't we get this to work with `String`/`Data`?